src/packageData.js

```diff
@@ -1,5 +1,6 @@
 const transformPlugins = {
   'babel-plugin-transform-async-to-generator': '@babel/plugin-transform-async-to-generator',
+  'babel-plugin-transform-async-to-module-method': '@babel/plugin-transform-async-to-generator',
```

Comment:
oh this an interesting one - technically it's that new package but we have to support adding options.. yay 😅 

```diff
{
  "plugins": [
-    ["@babel/transform-async-to-module-method"]
+    ["@babel/transform-async-to-generator", {
+      "module": "bluebird",
+      "method": "coroutine"
+    }]
  ]
}
```

Reply:
Oops, sorry I forgot the option 😭